### PR TITLE
feat: add example of executor using fastai

### PR DESCRIPTION
**Pull request by davidbp**
Add Executor leveraging fastai to encode images

### CODE REVIEW

**Review by Yongxuanzhang — COMMENTED** [review]
the title should be fastai?

**Comment by davidbp** [comment]
> the title should be fastai?

sure, thanks!

File: .github/2.0/cookbooks/Executor.md [hunk 1]
```diff
@@ -489,3 +489,36 @@ Line number corresponds to the 1.x code:
...
+        super().__init__(*args, **kwargs)
+        from fastai.vision.models import resnet18
+        
+        self.resnet = resnet18()
+
+     @requests
+     def encode(self, docs, **kwargs):
+         batch_images = []
+         for doc in docs:
+             batch_images.append(doc.blob)
```
[the remark this answers is not in the document]
Comment (davidbp) [reply .github/2.0/cookbooks/Executor.md:514]: We could, but .. is there any need or you mean it should be the default way to do it?

[the remark this answers is not in the document]
Comment (JoanFM) [reply .github/2.0/cookbooks/Executor.md:514]: let's do it since it will be shorter

[the remark this answers is not in the document]
Comment (davidbp) [reply .github/2.0/cookbooks/Executor.md]: done

File: .github/2.0/cookbooks/Executor.md [hunk 2]
```diff
@@ -489,3 +489,33 @@ Line number corresponds to the 1.x code:
...
+        super().__init__(*args, **kwargs)
+        from fastai.vision.models import resnet18
+        
+        self.model = resnet18()
+
+     @requests
+     def encode(self, docs, **kwargs):
+         batch_images = docs.get_attributes('blob')
+         batch = Torch.Tensor(batch_images)
+         batch_embeddings = self.model(batch)
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:514]: this does not make much sense.  Shouldn't this be
`
batch_embeddings = self.model(batch).detach().numpy()
for doc, embedding in zip(docs, batch_embeddings)

`

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — CHANGES_REQUESTED** [review]
I think also u can move the import of `fastai` on top. The need to hide the imports is not anymore there

File: .github/2.0/cookbooks/Executor.md [hunk 3]
```diff
@@ -563,3 +563,93 @@ class ResnetImageEncoder(Executor):
 
 ```
 
+=======
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md]: remove this line coming from some bad merge

**Review by JoanFM — APPROVED** [review]

**Review by JoanFM — APPROVED** [review]

File: .github/2.0/cookbooks/Executor.md [hunk 4]
```diff
@@ -532,8 +532,45 @@ Line number corresponds to the 1.x code:
...
 
+
+
 ## Executors in Action
 
+
+
+### Fastai
+
+This `Executor` uses the [ResNet18](https://docs.fast.ai) network for object classification on images provided by [fastai](https://github.com/fastai/fastai).
```
[the remark this answers is not in the document]
Comment (davidbp) [reply .github/2.0/cookbooks/Executor.md:543]: I like it more the way it is, it is not a label I populate. It is a vector of activations (same is done in the Pytorch Executor).

**Review by JoanFM — APPROVED** [review]
